extensions/default/src/Toolbar/ToolBoxWrapper.tsx

```diff
@@ -20,24 +20,34 @@ export function ToolBoxButtonGroupWrapper({ groupId, items, onInteraction, ...pr
           key={item.id}
           size="small"
           className={props.disabled && 'text-primary'}
-          onInteraction={() =>
-            onInteraction?.({ groupId, itemId: item.id, commands: item.commands })
-          }
+          onInteraction={() => {
```

Comment:
Can you preserve the event?  Otherwise it is difficult for some interactions to popup a localized menu etc.